test/powershell/Host/PSVersionTable.Tests.ps1

```diff
@@ -18,7 +18,9 @@ Describe "PSVersionTable" -Tags "CI" {
 
     }
     It "GitCommitId property should not contain an error" {
-       $PSVersionTable.GitCommitId | Should not match "powershell.version"
+       $powershellVersionFile = Join-Path -Path $PSScriptRoot -ChildPath "assets/powershell.version"
```

Comment:
I'm not sure if this test is really useful. The content contained in `powershell.version` is guaranteed (by the build script) to be the same as in `PSVersionInfo.generated.cs`. The only possible failure I can see for this test is that we are running tests outside of where powershell was being built and thus `powershell.vesrion` is not available.

I believe it's more useful to check if both `GitCommitId` and `PSVersion` are nonempty and `GitCommitId` starts with `PSVersion`, or something like that.